src/PrestaShopBundle/Translation/Translator.php

```diff
@@ -29,6 +29,9 @@
 
 use Symfony\Bundle\FrameworkBundle\Translation\Translator as BaseTranslator;
 
+/**
+ * Replaces the original Symfony FrameworkBundle translator
+ */
```

Comment:
I think you wanted to write "Replacement for the original Symfony FrameworkBundle translator" instead:

https://github.com/PrestaShop/PrestaShop/pull/14966#discussion_r310576557

Reply:
Right!